tests/boardfarm_plugins/boardfarm_prplmesh/prplmesh_config_compose.json

```diff
@@ -0,0 +1,55 @@
+{
+    "prplmesh_compose": {
+        "name": "agent",
+        "board_type": "prplmesh_compose",
+        "role": "agent",
+        "conn_cmd": "",
+        "devices": [
+            {
+                "name": "lan",
+                "type": "prplmesh_compose",
+                "role": "controller",
+                "delay" : 7,
+                "conn_cmd": ""
+            },
+            {
+                "name": "lan2",
+                "type": "prplmesh_compose",
+                "role": "agent",
+                "delay" : 30,
```

Comment:
This should have been 7 (or not there at all) I think, but it's something I did :grin:

Reply:
Fixed on d0e14154